### DISCUSSION TITLE
Official release of Windows binary

### DISCUSSION THREAD

**Discussion by otseng** [eliranwong/UniqueBible#1176] [Windows]
Official binary version of UBA for Windows now released:

https://github.com/eliranwong/UniqueBible/releases/tag/windows_31.62.1

**Comment by otseng** [comment]
Released 31.65 for Windows:

https://github.com/eliranwong/UniqueBible/releases/tag/windows_31.65

Fixed downloading dictionary and encyclopedia resources.

**Comment by otseng** [comment]
This has been deleted due to limitations with binary releases.

**Comment by passionate2023** [comment]
https://github.com/eliranwong/UniqueBible/releases/tag/windows_31.65 is not working

**Reply by otseng** [reply]
This has been deleted.

**Reply by onegladiator4u** [reply]
Is there a way to get old release ?

**Reply by otseng** [reply]
No, all the binary versions have been deleted.

**Reply by onegladiator4u** [reply]
> No, all the binary versions have been deleted.

@otseng ok brother
